third_party/blink/renderer/core/cobalt/performance/performance_extensions.cc

```diff
@@ -55,14 +57,23 @@ uint64_t PerformanceExtensions::measureUsedCpuMemory(ScriptState* script_state,
 
 ScriptPromise PerformanceExtensions::getAppStartupTime(
     ScriptState* script_state,
-    const Performance&,
+    const Performance& performance_obj,
     ExceptionState& exception_state) {
   auto* resolver = MakeGarbageCollected<ScriptPromiseResolver>(
       script_state, exception_state.GetContext());
   int64_t startup_time = 0;
   BindRemotePerformance(script_state)->GetAppStartupTime(&startup_time);
   ScriptPromise promise = resolver->Promise();
+#if BUILDFLAG(IS_STARBOARD)
+  int64_t time_origin = performance_obj.GetTimeOriginInternal().ToInternalValue();
```

Comment:
Instead of converting this to an int64_t, which is not an intuitive type, let's convert startup_time to a base::Time to make the math we're doing more clear.